pages/_app.jsx

```diff
@@ -18,14 +18,24 @@ export default function App({ Component, pageProps }) {
     setGames(games.filter((game) => game !== gameToRemove));
   }
 
+  function handleEditGame(gameToEdit) {
+    setGames(
+      games.map((game) => {
+        if (game.id === gameToEdit.id) return gameToEdit;
+        return game;
+      })
```

Comment:
```suggestion
      games.map((game) => game.id === gameToEdit.id ? gameToEdit : game)
```
👍 It's fine but here's a perfect usecase for the ternary